index.d.ts

```diff
@@ -21,6 +21,18 @@ interface LogDictionary {
 }
 
 declare module 'werelogs' {
+
+    class EndLogger {
```

Comment:
should it not be exported?

Reply:
We are not using it directly, it's returned by the `end()` method, so we do not need to export it.